snooty/postprocess.py

```diff
@@ -710,6 +713,120 @@ def enter_page(self, fileid_stack: FileIdStack, page: Page) -> None:
                 )
 
 
+class GuidesHandler(Handler):
+    """Constructs a dictionary of chapters and their data and returns metadata on individual guides."""
+
+    def __init__(self, context: Context) -> None:
+        super().__init__(context)
+        self.chapters: Dict[str, n.SerializedNode] = {}
+
+    def enter_node(self, fileid_stack: FileIdStack, node: n.Node) -> None:
+        def handleChapter(chapter: n.Directive) -> None:
```

Comment:
Python uses snake_case, not camelCase.

It's also better to just have these be methods of GuidesHandler: otherwise the compiler has to issue far more instructions to create the closures at runtime.